### ISSUE TITLE
composer update for laravel 5 does not work

### ISSUE THREAD

**Issue by cnweibo** [Zizaco/confide#538]
@Zizaco +1 According to readme, when added "zizaco/confide": "dev-5.0" and do composer update, it always say:  The requested package zizaco/confide could not be found in any version, there may be a typo in the package name.
I do not know what happens here.
Please help to fix it and public can use it as usual on laravel4.1 .

Thanks~!

**Comment by kageedwards** [comment]
@cnweibo Composer will look for the Packagist branch. I used `"zizaco/confide": "5.0.x-dev"`. Worked perfectly.

**Comment by mrgwilliam** [comment]
The 5.0.x-dev is currently rendering an error

**Comment by kageedwards** [comment]
Yeah, I just realized that as I finished the process on my other system. I put an issue in and included the temporary fix I did...
https://github.com/Zizaco/confide/issues/539

**Comment by cnweibo** [comment]
what i temp do is to use original laravel 4.x workable confide version and adapt to laravel5.1 manually.
